### ISSUE TITLE
Use Space+dots-24 chord to toggle image mode

### ISSUE THREAD

**Issue by alexmoon** [tactile-eng/cadendum#1]
For consistency between the keyboard and Braille shortcuts, we'd like to use Space+dots-24 to match the NVDA+I keyboard shortcut. That shortcut should be the default but it should be configurable in NVDA settings.
